tests/network/bindingplugin_passt.go

```diff
@@ -153,28 +133,37 @@ var _ = Describe(SIG(" VirtualMachineInstance with passt network binding plugin"
 					return nil
 				}
 
-				startClientVMI := func() {
-					clientVMI = libvmifact.NewAlpineWithTestTooling(
+				DescribeTable("Client server connectivity", func(ports []v1.Port, tcpPort int, ipFamily k8sv1.IPFamily) {
+					libnet.SkipWhenClusterNotSupportIPFamily(ipFamily)
+
+					By("starting a client VMI")
+					clientVMI := libvmifact.NewAlpineWithTestTooling(
 						libvmi.WithPasstInterfaceWithPort(),
```

Comment:
Unrelated to the PR: Why are we specifying a port on the client VMI's interface?